src/neuroconv/datainterfaces/behavior/deeplabcut/_dlc_utils.py

```diff
@@ -0,0 +1,285 @@
+import datetime
+import importlib
+import os
+import pickle
+import warnings
+from pathlib import Path
+from platform import python_version
+from typing import List, Optional, Union
+
+import cv2
```

Comment:
Ah, sure

Reply:
Good catch

Reply:
But as to why it was not throwing error, it's because we don't test global importability of this private submodule and it's not triggered by anything global (the import of this submodule is itself lazy in the interface usage)

Reply:
This is because of the protection with `__all__` on the `__init__` files?

Reply:
I didn't read much about the reasoning, only that it triggered on all the ones not contained in the `__all__` of that file (which is most if not all of them)